source/tests/test_endpoints.py

```diff
@@ -13,8 +13,13 @@ def test_get_comments():
     assert len(resp_json) > 0
 
 
+def test_get_comments_without_token():
+    response = TEST_CLIENT.get(COMMENTS)
+    assert response.status_code == HTTPStatus.BAD_REQUEST
+    assert 'Access token is required' in response.get_json()['error']
```

Comment:
This hard-coded error message might lead to issues later if we change it. Maybe make a constant for the message at the top that we can modify or make this a more general assertion?

Reply:
Sure! Thanks for the suggestion.